atlasdb-client-protobufs/build.gradle

```diff
@@ -6,7 +6,7 @@ buildscript {
     mavenCentral()
   }
   dependencies {
-    classpath 'com.google.protobuf:protobuf-gradle-plugin:0.7.6'
```

Comment:
Did you regenerate the .proto files. All the `*.Persistence` files should be regenerated and the changes if any committed along.